src/main/java/separator/Separator.java

```diff
@@ -0,0 +1,67 @@
+package separator;
+
+import java.util.ArrayList;
+
+public class Separator {
+    public static ArrayList<Character> separators = new ArrayList<>();
+
+    public Separator() {
+        separators.add(',');
+        separators.add(':');
+    }
```

Comment:
`static`으로 구분자가 선언되었기 때문에 스태틱 블록으로 묶는 것도 괜찮을..려나요?
```suggestion
    public static ArrayList<Character> separators = new ArrayList<>();

    static {
        separators.add(',');
        separators.add(':');
    }
```

Reply:
제가 자바 문법에 익숙치 않아서 기본 생성자로 작성했었습니다..ㅠ static 이라는 생성문법에 익숙치 않아서 기본 생성자로 작성했었습니다..ㅠ static에 대해서 공부해보겠습니다! 감사합니다!